src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  let line = '';
+  const wifeOrHusband = person.sex === 'm' ? 'wife' : 'husband';
+
+  if (!person.isMarried) {
+    line = 'I am not married';
+  } else {
+    line = `${person.partnerName} is my ${wifeOrHusband}`;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age
+        ? <p className="Person__age">{`I am ${person.age}`}</p>
+        : ''}
```

Comment:
```suggestion
      {person.age &&
         <p className="Person__age">{`I am ${person.age}`}</p>
       }
```

same but shorter

Reply:
Wrote it my way 'cause of incorrect work in case of '0' age - in this case it just prints 0 even instead of "I am 0".